.github/workflows/ci.yml

```diff
@@ -32,6 +32,11 @@ jobs:
             - name: Test with oldest-supported-numpy
               run: uv run --no-dev --with oldest-supported-numpy python -m unittest discover tests
 
+            - name: Lint
+              run: |
+                  uv run ruff check --fix --exit-non-zero-on-fix
```

Comment:
Is it equivalent to remove both arguments? Presumably we don't care about actually making changes to the files since we'll throw away the state after the test finishes

Reply:
Nice nice https://github.com/tpgillam/mt2/pull/80/commits/fd6df288d2969784aee6e0282ebd933c39a53325

Reply:
I have checked that this has the same return codes without both arguments.